tests/src/Behat/OeThemeTestContext.php

```diff
@@ -400,32 +407,17 @@ public function assertEuMobileLogo(string $language): void {
   /**
    * Asserts that the header logo contains all required attributes.
    *
-   * @Then the :language header logo should contain accessibility attributes
+   * @Then the header logo should contain accessibility attributes with link :link label :label alt :alt title :title
    */
-  public function assertHeaderLogoAccessibility(string $language): void {
-    $theme_name = \Drupal::theme()->getActiveTheme()->getName();
-    $component_library = \Drupal::config($theme_name . '.settings')->get('component_library');
-    $eu_official_languages = $this->getEuLanguages();
-
+  public function assertHeaderLogoAccessibility(string $link, string $label, string $alt, string $title): void {
     $link_selector = '.ecl-site-header-' . $this->getEclBranding() . '__logo-link';
-    $link = $this->getSession()->getPage()->find('css', $link_selector);
-    if ($component_library === 'ec') {
-      $href = !empty($eu_official_languages[$language]) ? 'https://ec.europa.eu/info/index_' . $eu_official_languages[$language] : 'https://ec.europa.eu/info';
-      $aria_label = t('Home - European Commission')->__toString();
-      $img_alt = t('European Commission logo')->__toString();
-      $img_title = t('European Commission')->__toString();
-    }
-    else {
-      $href = !empty($eu_official_languages[$language]) ? 'https://european-union.europa.eu/index_' . $eu_official_languages[$language] : 'https://european-union.europa.eu';
-      $aria_label = t('Home - European Union')->__toString();
-      $img_alt = t('European Union flag')->__toString();
-      $img_title = t('European Union')->__toString();
-    }
-    Assert::assertEquals($href, $link->getAttribute('href'));
-    Assert::assertEquals($aria_label, $link->getAttribute('aria-label'));
-    $img = $link->find('css', 'img');
-    Assert::assertEquals($img_alt, $img->getAttribute('alt'));
-    Assert::assertEquals($img_title, $img->getAttribute('title'));
+    $logo_link = $this->getSession()->getPage()->find('css', $link_selector);
+
+    Assert::assertEquals($link, $logo_link->getAttribute('href'));
+    Assert::assertEquals($label, $logo_link->getAttribute('aria-label'));
+    $img = $logo_link->find('css', 'img');
+    Assert::assertEquals($alt, $img->getAttribute('alt'));
+    Assert::assertEquals($title, $img->getAttribute('title'));
```

Comment:
Why do we need to change the existing assertions since the info is the same for each language?